src/select/dropdown.js

```diff
@@ -80,6 +80,26 @@ export default class SelectDropdown extends React.Component<DropdownPropsT> {
     e.preventDefault();
   };
 
+  getHighlightedIndex = () => {
+    const {value, options, valueKey} = this.props;
+    // Highlight only first value as menu supports only a single highlight index
+    let firstValue: OptionT = {};
+
+    if (Array.isArray(value) && value.length > 0) {
```

Comment:
I believe the multi select filters options that have already been selected. What situation were you thinking about that required this code path?

Reply:
this check is inspired from the getItemLabel function in same file